### PR TITLE
ENYO-5726: Reduced onScrollStart and onScrollStop calls on key down

### PR DESCRIPTION
**Pull request by 0x64**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed

* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
After removing animations during scroll by arrow key down,
onScrollStart and onScrollStop are called back on every focus movement.
If an app sets at least one of those callbacks, this behavior may affect the performance.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
To reduce unnecessary callbacks,
this PR changes lists to fire onScrollStop after waiting a certain period of time
from the last update of the scroll position.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
ENYO-5726

### CODE REVIEW

File: packages/ui/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 The following is a curated list of changes in the Enact ui module, newest changes on the top.
 
+## [unreleased]
+
+### Changed
+- `ui/VirtualList`, `ui/VirtualGridList`, and `ui/Scroller` components not to call `onScrollStop` too frequently during scroll
```
Comment (viodragon2) [inline packages/ui/CHANGELOG.md]: `ui/Scroller` should be inclusive of `ui/VirtualList*`. Plus, it's missing the context for "non-animation" only case.

I suggest something like

> - non animating `ui/Scroller` to debounce `onScrollStop` events

File: packages/ui/Scrollable/Scrollable.js [hunk 2]
```diff
@@ -930,6 +944,7 @@ class ScrollableBase extends Component {
 			this.scrolling = true;
 			this.forwardScrollEvent('onScrollStart');
 		}
+		this.scrollStopJob.start();
```
[the remark this answers is not in the document]
Comment (viodragon2) [reply packages/ui/Scrollable/Scrollable.js]: After discussing w/ @0x64 a bit, I think this should be changed to `this. scrollStopJob.stop()`

[the remark this answers is not in the document]
Comment (0x64) [reply packages/ui/Scrollable/Scrollable.js]: Thanks for pointing out!

File: packages/ui/CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,11 @@
 
 The following is a curated list of changes in the Enact ui module, newest changes on the top.
 
+## [unreleased]
+
+### Changed
```
[the remark this answers is not in the document]
Comment (0x64) [reply packages/ui/CHANGELOG.md]: Fixed to `Fixed`

File: packages/ui/Scrollable/Scrollable.js [hunk 4]
```diff
@@ -1011,7 +1026,10 @@ class ScrollableBase extends Component {
 		}
 
 		this.childRef.setScrollPosition(this.scrollLeft, this.scrollTop, dirX, dirY);
-		this.forwardScrollEvent('onScroll');
+		if (this.scrolling) {
+			this.forwardScrollEvent('onScroll');
+			this.scrollStopJob.start();
```
[the remark this answers is not in the document]
Comment (0x64) [reply packages/ui/Scrollable/Scrollable.js]: You're right. Rolled back.

File: packages/ui/CHANGELOG.md (hunk 1 above)
Comment (0x64) [reply packages/ui/CHANGELOG.md]: We normally write all components. I updated the description but still I hope someone changes it to more fluent description.

**Pull request merged**